src/plugins/blocksmith/components/youtube-playlist-block-modal/getYoutubePlaylistIds.js

```diff
@@ -0,0 +1,29 @@
+export const YOUTUBE_PLAYLIST_ID_QUERY_STRING_REGEX = /list=[a-zA-Z0-9_-]+/;
+export const YOUTUBE_VIDEO_ID_QUERY_STRING_REGEX = /v=[a-zA-Z0-9_-]+/;
+export const ID_REGEX = /[a-zA-Z0-9_-]/;
+
+const getYoutubePlayId = (str) => {
```

Comment:
@efjacobson @jalopez13 - can we match the constant name to the filename, i.e. `getYoutubePlaylistId`. Also, why is the filename plural? Please add docBlock to the function.

Reply:
@arunkarnati its because it can return both a video id and a playlist id. ill add a docblock

Reply:
@arunkarnati added